src/components/album/AlbumsList.js

```diff
@@ -145,7 +145,7 @@ const AlbumsList = () => {
   };
 
   return (
-    <Container>
+    <Container fullWidth>
```

Comment:
`Container` component does not have a `fullWidth` property. Maybe you want to use `maxWidth` ? [https://mui.com/material-ui/api/container/#container-prop-maxWidth](https://mui.com/material-ui/api/container/#container-prop-maxWidth)

Currently this is causing this error in the console:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/a1545ea5-43ec-4c8c-9d2f-721f83056a9f)

Reply:
@akosasante thank you, Akos! I was trying to remove extra spaces from the right and left side of the page (what we can see on the image below) and I´ve been trying many things, but **maxWidth** prop you suggested, worked out! thank you! 
Please tell me if it´s not looking good now because I´m not sure..
![image](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/103755090/2e37e46b-1475-4c52-b170-c7478d66e68d)